src/main/sensors/voltage.c

```diff
@@ -194,7 +199,8 @@ void voltageMeterADCInit(void)
         voltageMeterADCState_t *state = &voltageMeterADCStates[i];
         memset(state, 0, sizeof(voltageMeterADCState_t));
 
-        pt1FilterInit(&state->filter, pt1FilterGain(GET_BATTERY_LPF_FREQUENCY(batteryConfig()->vbatLpfPeriod), HZ_TO_INTERVAL(50)));
+        pt1FilterInit(&state->filter, pt1FilterGain(GET_BATTERY_LPF_FREQUENCY(batteryConfig()->vbatDisplayLpfPeriod), HZ_TO_INTERVAL(50)));
+        pt1FilterInit(&state->sagFilter, pt1FilterGain(GET_BATTERY_LPF_FREQUENCY(batteryConfig()->vbatSagLpfPeriod), HZ_TO_INTERVAL(200)));
```

Comment:
This shouldn't be 200hz. The `TASK_BATTERY_VOLTAGE` task runs at 50hz and this is the sample rate from the voltage sensor.

Reply:
I changed `TASK_BATTERY_VOLTAGE` to run at 200hz, as I do need the battery sag compensation voltage input to be sampled more frequently than 50hz... in tasks.c 381:
```    [TASK_BATTERY_VOLTAGE] = DEFINE_TASK("BATTERY_VOLTAGE", NULL, NULL, batteryUpdateVoltage, TASK_PERIOD_HZ(200), TASK_PRIORITY_MEDIUM),
```

Reply:
Then you need to change the other filters to be initialized at 200hz sample period. Can have the sample period arbitrarily chosen. Also simply increasing a task frequency 4 times is not something that should be done without more impact analysis. For example have you done any research to see what impact this has on loop stability - particularly for the borderline F411?

Reply:
I have F411's and didn't see any specific issue with 200hz.  

If battery sampling causes a big loop time hit every 20ms, increasing that to every 5ms would be undesirable, for sure.  However if it was as bad enough to be a problem every 5ms, it should already appear as an issue every 20ms, I guess.

What would be the best way to assess this?